test-bench-utils/lib/sinks/xxe.js

```diff
@@ -18,7 +22,13 @@ module.exports['libxmljs.parseXmlString'] = async function parseXmlString(
 ) {
   if (noop) return 'NOOP';
 
-  const result = libxmljs.parseXmlString(input, { noent: !safe });
+  let result;
```

Comment:
1.  Is this what we said we'd do when we groomed this ticket?  I can't remember.  Obviously the idea was to fix the crash, but this seems a bit strange to me to "fake" that the test passes when really we're just changing what the test is doing.  Personally I'd almost prefer just not calling anything in this case and have these doc'ed as expected failures for node 14.  but i don't feel that strongly about it.

2.  If we do keep this it feels like it could be shortened quite a bit.  could just re-work where things are required to be:

```
libxmljs2 = require('libxmljs2');
try {
   libxmljs = require('libxmljs');
} catch {
   // workaround for lack of prebuilts on node14
   libxmljs = libxmljs2;
}
```

and then we could just leave the original code here as is.